pyEpiabm/pyEpiabm/intervention/social_distancing.py

```diff
@@ -0,0 +1,60 @@
+#
+# Social distancing Class
+#
+
+import random
+
+from pyEpiabm.core import Parameters
+from pyEpiabm.intervention import AbstractIntervention
```

Comment:
```suggestion
from pyEpiabm.core import Parameters

from .abstract_intervention import AbstractIntervention
```

check the local import works, but I think we settled on this style for imports within the same submodule. Its probably not an issue but can help you avoid circular import errors later down the line